src/app/store.ts

```diff
@@ -1,6 +1,10 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
 
-const rootReducer = combineSlices();
+import { todosSlice } from '../features/todos/todos';
+import { filterSlice } from '../features/filter/filter';
+import { currentTodoSlice } from '../features/currentTodo/currentTodo';
+
+const rootReducer = combineSlices(todosSlice, filterSlice, currentTodoSlice);
```

Comment:
The function `combineSlices` is not a valid function from '@reduxjs/toolkit'. You should use `combineReducers` to combine the slices into a root reducer.